XiEditor/EditView.swift

```diff
@@ -198,7 +209,7 @@ class EditView: NSView, NSTextInputClient {
             let randcolor = NSColor(colorLiteralRed: Float(drand48()), green: Float(drand48()), blue: Float(drand48()), alpha: 1.0)
             attrString.addAttribute(NSForegroundColorAttributeName, value: randcolor, range: NSMakeRange(0, s.utf16.count))
             */
-            styleMap?.applyStyles(text: s, string: attrString, styles: line.styles, selColor: selcolor())
+            styleMap?.applyStyles(text: s, string: attrString, styles: line.styles, selColor: textSelectionColor)
```

Comment:
Since we're drawing the selection rectangles rather than adding them as bgcolor attributes to the string, there's no need to pass the selection color into applyStyles.